_posts/2021-01-29-design-system-schizophrenia.md

```diff
@@ -0,0 +1,110 @@
+developers code single-use ultra-specific components, when there is all that but
+you do nothing to fix it then, he says, you're *not* building a Design System.
+
+And I strongly agreee with him.
+
+He ends his article with this powerful schema who have help me grow a strong
+opinion on the situation.
+
+![A diagram showing on one side an opposition between code and design, and on the other a cycle between features and a Design System](/assets/img/design-system-schizophrenia-diagram.png)
+
+Yeah. Unfortunately installing a DS means *change*. We don't like change. It
+means switching from the traditional "designers vs developers" mindset (left) to
+a Product mindset (right). We don't like switching mindset.
+
+To install a  Design System what you want to target is having a *team* that
+owns a *product* (the DS and its artifacts) that have *customers* (your others
+teams).
+
+But, I'm forced to admit that I don't see a lot of Design System *teams* in
+companies.
+
+There are Design System *projects*, for sure. It's very trendy those days. They
+are mainly driven by the base of the organization, ie. some designers and/or
+developers. They work on it on the week-end. They come early in the morning
+or they leave late at night to have some time to code for the Design System
+besides their "real" job, shipping features.
+
+Sometimes, when those Design System projects begin to see some traction
+(because, yeah, it's a powerful concept) and when the initial volunteers haven't
+burned out yet, leadership enter the game and declare "We have a Design
+System! It's so good! Thanks so and so.".
+They write a blog post showing the world how their Design System revolutionized
+their company while in fact nothing happen inside.
+
+Nothing happen inside because they haven't embraced that a Design System is a
+product that need a dedicated team.
+
+It's far from impossible to form such a team.
+We already have infrastructure/platform teams that give others a solid platform
+to run their services. We have security teams that give others tools to build
+secure services. We have engineering productivity teams that help others with
+day to day tools (CI, testing environment, etc.). We have analysts and UX
+researchers to support others at making decisions based on data.
+
+We have all those teams but you can't form one for the Design System? What's
+your problem?
+
+Is it that you have to group designers and developers together and that's new
+for you?
+Don't you already do that for feature teams, squads, swarms or whatever name you
+give to your multidiciplinary teams? Those that work on features.
+
+Is it that you don't want to start a war by having to decide whether this team
+reports to the VP Product or the VP Engineering? Does it really matter?
+
+Is it because your designers or developers are already understaffed? Don't you
+see how such a organization could be exactly what you need to accomodate with
+that?
```

Comment:
```suggestion
Is it because designers or developers are already understaffed? Well shoot, that's precisely why such a team ultimately building a Design System would help: it's there to help you do more with less.
```